content/blog/model-calibration/index.Rmd

```diff
@@ -0,0 +1,411 @@
+---
+output: hugodown::hugo_document
+
+slug: model-calibration
+title: Model Calibration
+date: 2022-11-17
+author: Edgar Ruiz
+description: >
+    Model Calibration is coming to tidymodels. This post covers the new plotting
+    functions, and our plans for future enhancements. 
+
+photo:
+  url: https://unsplash.com/photos/s3B_pjK7UIs
+  author: Graphic Node
+
+categories: [package]
+tags: [model, plots]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  #eval = FALSE
+)
+
+library(probably)
+library(dplyr)
+library(ggplot2)
+library(tidymodels)
+```
+
+I am very excited to introduce work currently underway.  We are looking to create 
+early awareness, and to receive feedback from the community. That is why the 
+enhancements discussed here are not yet in CRAN. 
+
+Even though the article is meant to introduce new package functionality. We also
+have the goal of introducing model calibration conceptually. We want to provide
+sufficient background for those who may not be familiar with model calibration.
+If you are already familiar with this technique, feel free to skip to the 
+[Setup](#example-data) section to get started. 
+
+To install the version of probably used here: 
+
+```{r, eval = FALSE}
+remotes::install_github("tidymodels/probably")
+```
+
+
+## Model Calibration
+
+*The goal of model calibration is to ensure that the estimated class probabilities 
+consistent with what would naturally occur.* If a model has poor calibration, 
+we might be able to post-process the original predictions to coerce them to have
+better properties. 
+
+There are two main components to model calibration:
+
+- **Diagnosis** - Figuring out how well the original (and re-calibrated) probabilities 
+perform.
+- **Remediation** - Adjusting the original values to have better properties.
+
+This post has details on diagnosing issues. 
```

Comment:
When reading this sentence I was asking myself if that was because that's what you've implemented so far. I think this section could nicely be combined with the next one.